internal/configure.go

```diff
@@ -314,19 +358,27 @@ func ConfigureCrypterForSpecificConfig(config *viper.Viper) crypto.Crypter {
 		tracelog.WarningLogger.Printf(DeprecatedExternalGpgMessage)
 		return openpgp.CrypterFromKeyRingID(keyRingID, loadPassphrase)
 	}
+	return nil
+}
 
-	if config.IsSet(CseKmsIDSetting) {
-		return awskms.CrypterFromKeyID(config.GetString(CseKmsIDSetting), config.GetString(CseKmsRegionSetting))
+func configureEncryptedPgpCrypter(config *viper.Viper) crypto.Crypter {
+	if !config.IsSet(YcKmsKeyIDSetting) {
+		tracelog.ErrorLogger.Fatalf("Yandex Cloud KMS key for client-side encryption and decryption must be configured")
 	}
 
-	if config.IsSet(YcKmsKeyIDSetting) {
-		return yckms.YcCrypterFromKeyIDAndCredential(config.GetString(YcKmsKeyIDSetting), config.GetString(YcSaKeyFileSetting))
-	}
+	yckmsEnveloper := yckmsenvlpr.EnveloperFromKeyIDAndCredential(
+		config.GetString(YcKmsKeyIDSetting), config.GetString(YcSaKeyFileSetting),
```

Comment:
maybe it's worth having separate yckms config for encrypted gpg to avoid misconfiguration..